Dockerfile

```diff
@@ -7,25 +9,33 @@ ARG BASE_IMAGE=quay.io/centos/centos:stream9
 FROM $BASE_IMAGE AS ironic-builder
 
 ARG IPXE_COMMIT_HASH=119c415ee47aaef2717104fea493377aa9a65874
+ARG MAKEFLAGS="-j100"
 
-RUN dnf install -y gcc git make xz-devel
+SHELL ["/bin/bash", "-ex", "-o", "pipefail", "-c"]
+RUN dnf install -y gcc make git xz-devel
 
-WORKDIR /tmp
+WORKDIR /tmp/ipxe/src
 
-RUN git clone https://github.com/ipxe/ipxe.git && \
-      cd ipxe && \
-      git reset --hard $IPXE_COMMIT_HASH && \
-      cd src && \
-      ARCH=$(uname -m | sed 's/aarch/arm/') && \
-      # NOTE(elfosardo): warning should not be treated as errors by default
-      NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
+RUN curl -Lf https://github.com/ipxe/ipxe/archive/${IPXE_COMMIT_HASH}.tar.gz | \
+       tar -zx --strip-components=1 -C /tmp/ipxe && \
+    ARCH=$(uname -m | sed 's/aarch/arm/') && \
+    # NOTE(elfosardo): warning should not be treated as errors by default
+    NO_WERROR=1 make bin/undionly.kpxe "bin-$ARCH-efi/snponly.efi"
 
 COPY prepare-efi.sh /bin/
 RUN prepare-efi.sh centos
 
 FROM $BASE_IMAGE
 
-ENV PKGS_LIST=main-packages-list.txt
+LABEL org.opencontainers.image.url="https://github.com/metal3-io/ironic-image"
+LABEL org.opencontainers.image.title="Metal3 Ironic Container"
+LABEL org.opencontainers.image.description="Container image to run OpenStack Ironic as part of Metal³"
+LABEL org.opencontainers.image.documentation="https://github.com/metal3-io/ironic-image/blob/main/README.md"
```

Comment:
Documentation would maybe better point to our user-guide?

Reply:
I agree,  let's point to the  user guide